frontend/resourceadm/pages/DeployResourcePage/DeployResourcePage.tsx

```diff
@@ -198,19 +198,29 @@ export const DeployResourcePage = ({
       case 'pending': {
         return (
           <div>
-            <Spinner size='xlarge' variant='interaction' title={t('resourceadm.deploy_spinner')} />
+            <StudioSpinner
+              size='xl'
+              variant='interaction'
+              spinnerTitle={t('resourceadm.deploy_spinner')}
+            />
           </div>
         );
       }
       case 'error': {
         return (
-          <Alert severity='danger'>
-            <Paragraph>{t('general.fetch_error_message')}</Paragraph>
-            <Paragraph>{t('general.error_message_with_colon')}</Paragraph>
-            {publishStatusError && <ErrorMessage>{publishStatusError.message}</ErrorMessage>}
-            {validatePolicyError && <ErrorMessage>{validatePolicyError.message}</ErrorMessage>}
-            {validateResourceError && <ErrorMessage>{validateResourceError.message}</ErrorMessage>}
-          </Alert>
+          <StudioAlert severity='danger'>
+            <StudioParagraph>{t('general.fetch_error_message')}</StudioParagraph>
+            <StudioParagraph>{t('general.error_message_with_colon')}</StudioParagraph>
+            {publishStatusError && (
+              <StudioErrorMessage>{publishStatusError.message}</StudioErrorMessage>
+            )}
+            {validatePolicyError && (
+              <StudioErrorMessage>{validatePolicyError.message}</StudioErrorMessage>
+            )}
+            {validateResourceError && (
+              <StudioErrorMessage>{validateResourceError.message}</StudioErrorMessage>
+            )}
```

Comment:
Should we add sizes to these texts as well so make sure they have the same size as the rest of the application? 😄